examples/example-gcp-observability/src/main/java/io/grpc/examples/observabilityHelloWorld/ObservabilityHelloWorldClient.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2023 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.observabilityHelloWorld;
```

Comment:
The original doc suggested `io.grpc.examples.helloworld` as the package and I can see it is the same as the base helloworld example package. But:

- do we need the leaf package to be `observabilityHelloWorld` or can it be just `observability` to keep it short?
- for the classes also you used the prefix `Observability` so that is another distinguishing factor (just in case we don't rename the package and put all of the classes in one jar file)

What I would prefer is to rename the leaf package to just `observability` and keep the class names as `HelloWorldClient` and `HelloWorldServer` but I would like to know what @ejona86 thinks.